### PR TITLE
Add Organizers' slides for 2018-12-17

### PR DESCRIPTION
**Pull request by marcoala**

### CODE REVIEW

File: organizers/2018-12-17.html [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+    <h1>💰 2018 Sponsors</h1>
+    <h2>Thanks for making the meetup possible!</h2>
+    <ul>
+        <li>Festicket</li>
+        <li>Funding Options</li>
+        <li>Thread</li>
+        <li>Time Out</li>
+        <li>Verve</li>
+        <li>Skills Matter</li>
+        <li>JetBrains</li>
```
Comment (adamchainz) [inline organizers/2018-12-17.html]: Alphabetically sort this list to show we don't give preference to any sponsor

File: organizers/2018-12-17.html [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+
+<section>
+    <h1>📅  2018 Year Review</h1>
+    <h2>New organisers</h2>
+    <ul>
+        <li>Marco Alabruzzo</li>
+        <li>Adam Johnson</li>
+        <li>James Pacileo</li>
+        <li>David Seddon</li>
+        <li>Çağıl Ulusahin</li>
```
Comment (adamchainz) [inline organizers/2018-12-17.html]: Mention Craig!

File: organizers/2018-12-17.html [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+    <h2>We want more speaker diversity</h2>
+    <h2>👩🏻‍💻 👨🏽‍💻 👩🏿‍💻 👨🏻‍💻 👩🏽‍💻 👨🏿‍💻 </h2>
+    <p>In 2018 only 2 female speakers: Lili Kastilio and Luna Chen</p>
+    <p>Long term goal is to have 50/50</p>
+</section>
+
+
+<section>
+    <h1>📅  2018 Year Review</h1>
+    <h2>New organisers</h2>
```
Comment (adamchainz) [inline organizers/2018-12-17.html]: James, David, and myself weren't new this year, so perhaps just "Organisers" as the title?

File: organizers/2018-12-17.html [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+        <li>We are looking for engaged sponsors</li>
+    </ul>
+</section>
+
+
+<section>
+    <h1>💰 New sponsors</h1>
+    <h2>Why sponsors the meetup?</h2>
+    <ul>
+        <li>Allow us to keep the metup runnign and growing</li>
```
Comment (adamchainz) [inline organizers/2018-12-17.html]: *running

File: organizers/2018-12-17.html [hunk 5]
```diff
@@ -0,0 +1,219 @@
...
+        <li>We could do more than just food</li>
+
+        <li>Cheap recruitment</li>
+        <ul>
+            <li>Unicorn hunt £240 monthly</li>
+            <li>Stack over flow jobs $489 monthly</li>
```
Comment (adamchainz) [inline organizers/2018-12-17.html]: Stack Overflow

File: organizers/2018-12-17.html [hunk 6]
```diff
@@ -0,0 +1,219 @@
...
+        <li>3. The notes get placed in a box</li>
+        <li>4. Draw commences</li>
+    </ul>
+</section>
+
+
+<section>
+    <h1>🎤 Talks</h1>
+    <p>
+        <strong>Django, MQTT & Websockets -Talk about Performance!</strong><br>
```
Comment (adamchainz) [inline organizers/2018-12-17.html]: space after "-"

File: organizers/2018-12-17.html (hunk 2 above)
Comment (marcoala) [reply organizers/2018-12-17.html]: Done, and reordered alphabetically

**Comment by marcoala** [comment]
@adamchainz can you merge this one?

File: code-of-conduct.md [hunk 7]
```diff
@@ -5,7 +5,7 @@ layout: default
 ## Be Awesome to Each Other
 
 The mission of [The London Django Meetup Group](http://www.djangolondon.com/)
-is to provide fun, welcoming and professional environments so that diverse
+is to provide a fun, welcoming and professional environments so that diverse
```
Comment (adamchainz) [inline code-of-conduct.md]: undo this change - there shouldn't be an "a" because it's talking about multiple environment*s*

**Comment by marcoala** [comment]
@adamchainz all sorted!

**Comment by adamchainz** [comment]
@cgl presume you were asking about adding the other page to the site?

**Pull request merged**